src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java

```diff
@@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
         return val;
     }
 
+    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
+      return a.getReadGroup() == b.getReadGroup() &&
+          a.getTile() == b.getTile() &&
+          // There used to not be an abs here, not sure why.
```

Comment:
Also, regarding == vs .equals, would you be ok with an or clause for .equals? Dataflow does serialization of classes and it is messing up the ==.